plugins/inputs/statsd/statsd.go

```diff
@@ -439,17 +460,16 @@ func (s *Statsd) udpListen(conn *net.UDPConn) error {
 		case <-s.done:
 			return nil
 		default:
-			n, _, err := conn.ReadFromUDP(buf)
+			n, addr, err := conn.ReadFromUDP(buf)
 			if err != nil && !strings.Contains(err.Error(), "closed network") {
-				log.Printf("E! Error READ: %s\n", err.Error())
+				log.Printf("E! [inputs.statsd] Error READ: %s\n", err.Error())
 				continue
 			}
 			b := s.bufPool.Get().(*bytes.Buffer)
 			b.Reset()
 			b.Write(buf[:n])
-
 			select {
-			case s.in <- b:
+			case s.in <- input{Buffer: b, Time: time.Now(), Addr: addr.IP.String()}:
```

Comment:
Noticed this when checking into the toml option name.  This happens when stopping the plugin.

```
2019-05-10T20:23:23Z I! [inputs.statsd] Statsd UDP listener listening on:  0.0.0.0:8125
2019-05-10T20:23:23Z I! [inputs.statsd] Started the statsd service on :8125
^C2019-05-10T20:23:27Z I! [inputs.statsd] Stopping the statsd service
2019-05-10T20:23:27Z I! Stopped Statsd listener service on  :8125
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x10 pc=0x16f7a20]

goroutine 8 [running]:
github.com/influxdata/telegraf/plugins/inputs/statsd.(*Statsd).udpListen(0xc0004d4380, 0xc0001e2008, 0xc0004d4408, 0x0)
        /home/dbn/src/influx/src/github.com/influxdata/telegraf/plugins/inputs/statsd/statsd.go:472 +0x250
github.com/influxdata/telegraf/plugins/inputs/statsd.(*Statsd).Start.func2(0xc0004d4380, 0xc0001e2008)
        /home/dbn/src/influx/src/github.com/influxdata/telegraf/plugins/inputs/statsd/statsd.go:378 +0x64
created by github.com/influxdata/telegraf/plugins/inputs/statsd.(*Statsd).Start
        /home/dbn/src/influx/src/github.com/influxdata/telegraf/plugins/inputs/statsd/statsd.go:376 +0x961